arch/arm/core/aarch64/Kconfig

```diff
@@ -112,6 +112,9 @@ config EXCEPTION_DEBUG
 	  and parameters, at a cost of code/data size for the human-readable
 	  strings.
 
+config XIP
+	select AARCH64_IMAGE_HEADER
```

Comment:
you still don't want to set this to default, right @carlocaione ?

Reply:
(I think this is default y for aarch32)

Reply:
You mean XIP? I don't think that in the ARM64 case this is very common since in most cases you have some kind of firmware running in EL3 / EL2 so I'd leave this as `n` by default.